README.md

```diff
@@ -3,29 +3,21 @@
 
 # 📤🐇 Hutch
 
-**Hutch** is an application stack for **Federated Data Discovery**.
+**Hutch** is an application stack for **Federated Analytics**.
```

Comment:
I would generalise this more. We don't want to be pigeon holed into Analytics _or_ Data Discovery.

```suggestion
**Hutch** is an application stack for **Federated Activities**, such as Analytics, Data Discovery or Machine Learning.
```